### ISSUE TITLE
[Bug]: truncate table internal table not reported error

### ISSUE THREAD

**Issue by heni02** [matrixorigin/matrixone#6087]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Environment

```Markdown
- Version or commit-id (e.g. v0.6.0 or 8b23a93):72a968f3e136442a164676d8b89b2233013f38a0
- Hardware parameters:
- OS type:
- Others:
```

### Actual Behavior

![image](https://user-images.githubusercontent.com/113406637/198824919-406f6d14-07fb-45fb-81a1-4228e78b05a6.png)
and same error other internal database tables

### Steps to Reproduce

```Markdown
use mo_catalog;
truncate table mo_account;
truncate table mo_role;
truncate table mo_user_grant;
truncate table mo_role_grant;
truncate table mo_role_privs;
```

**Comment by daviszhen** [comment]
it was fixed at #6098

**Comment by heni02** [comment]
confirm,closed
commit id:249f56400184a0828afdc615c8b5d75e8c57cecc
![image](https://user-images.githubusercontent.com/113406637/199638231-be718f59-46af-48b0-a587-7ad5efdd2481.png)
